network/src/protocols/wire/handshake/v1/mod.rs

```diff
@@ -39,6 +40,8 @@ pub enum ProtocolId {
     StateSyncDirectSend = 3,
     DiscoveryDirectSend = 4,
     HealthCheckerRpc = 5,
+    ConsensusRpcCbor = 6,
+    ConsensusDirectSendCbor = 7,
```

Comment:
You mentioned in the top about changing back afterwards.  Wouldn't you need to do this every time you want to update the protocol?

Reply:
yeah, if the performance is okayish, we don't need to switch back, switching could be config based instead of hard coded like today